src/redux/books/books.js

```diff
@@ -0,0 +1,41 @@
+const ADD_BOOK = 'Bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'Bookstore/books/REMOVE_BOOK';
+
+const initialState = [{
+  id: '1',
+  title: 'Romeo and Juliet',
+  author: 'William Shakeaspeare',
+},
+{
+  id: '2',
+  title: 'La vida es sueño',
+  author: 'Calderon de la Barca',
+},
+];
```

Comment:
- [ ] As requested in the project description, please set the initial state here to an empty array.